### PR TITLE
Initial packaging updates

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

File: Src/IronPythonConsoleAny/IronPythonConsoleAny.csproj [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(ProjectDir)..\..\Build\Common.proj" />
   <PropertyGroup>
-    <TargetFrameworks>net40;net45</TargetFrameworks>
-    <TargetFrameworks Condition="'$(BuildNetCoreApp)' == 'true'">$(TargetFrameworks);netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetFrameworks)' == ''">net40;net45</TargetFrameworks>
```
Comment (slozier) [inline Src/IronPythonConsoleAny/IronPythonConsoleAny.csproj]: Missing netcoreapp2.0.

File: appveyor.yml [hunk 2]
```diff
@@ -78,16 +78,17 @@ test_script:
 #      artifacts configuration    #
 #---------------------------------#
 
-#artifacts:
```
Comment (slozier) [inline appveyor.yml]: Did you mean to remove this or just uncomment it?

File: Src/IronPythonConsoleAny/IronPythonConsoleAny.csproj (hunk 1 above)
Comment (slide) [reply Src/IronPythonConsoleAny/IronPythonConsoleAny.csproj]: I didn't think we needed to build the console for netcoreapp2.0. Do we?

File: appveyor.yml (hunk 2 above)
Comment (slide) [reply appveyor.yml]: I have a fix for this, I noticed after the initial push

**Comment by slozier** [comment]
Close and reopen to see if test_subprocess passes.

**Pull request merged**